README.md

```diff
@@ -84,4 +84,15 @@ The site is statically generated using the [hugo](https://gohugo.io/) framework.
 3. Open a terminal in the root directory of the developer-glossary project and run `hugo server -D`
 4. Check that no errors appear in the terminal, it should tell you it is running on localhost:1313. That means you can view the site running locally by going to [http://localhost:1313](http://localhost:1313) in your browser.
 
-You're off to the races! Reference [hugo docs](https://gohugo.io/about/) to learn more and start making code contributions.
\ No newline at end of file
+You're off to the races! Reference [hugo docs](https://gohugo.io/about/) to learn more and start making code contributions.
+
+## Language use guidelines for contributions
+
+All contributions made towards adding a new term and its definition must adhere to these common guidelines:
+
+1. Every contribution made should adhere to the [Code of Conduct](https://github.com/do-community/developer-glossary/blob/main/code_of_conduct.md)
+2. Avoid usage of country specific language when adding a new term, in the term definition and for the examples provided.
+3. Use simple language. Avoid idioms and be clear.
```

Comment:
Can you  link "Use simple language" to here: https://plainlanguage.gov/resources/articles/dash-writing-tips/

Reply:
Yeah sure.

Reply:
@ruf-io I have made the change